_dashboards/management/scheduled-query-acceleration

```diff
@@ -0,0 +1,228 @@
+---
+layout: default
+title: Optimizing direct queries from OpenSearch Service to Amazon S3 with Scheduled Query Acceleration
+parent: Data sources
+nav_order: 18
+has_children: true
+---
+
+# Optimizing direct queries from OpenSearch Service to Amazon S3 with Scheduled Query Acceleration
+Introduced 2.17
+{: .label .label-purple }
+
+## Overview
+
+Scheduled Query Acceleration (SQA) is designed to enhance user experience by addressing the challenges often faced with managing and refreshing indexes, views, and data in an automated way. This guide will help you understand how SQA can improve workflows, reduce costs, and enhance transparency in managing your data.
+
+## Key Benefits
+
+### Cost Reduction Through Optimized Resource Usage
+
+By optimizing the driver node utilization, SQA significantly reduces the costs linked to maintaining auto-refresh capabilities for indexes and views. With fewer continuous driver operations needed, you can expect more efficient resource usage and a lower impact on your budget.
+
+For example, by comparing the total cost of using an internal scheduler versus an external scheduler at different refresh intervals (e.g., 5 minutes, 15 minutes, 30 minutes), SQA shows significant cost advantages.
+
+### Improved Observability of Refresh Operations
+
+SQA enhances transparency around index states and refresh operations, allowing you to see when refreshes occur. This observability provides improved insights into data processing, making it easier to understand the state of your data in real time. This feature helps in reducing uncertainty around the "refreshing" status, enabling data-driven decision-making based on actual system state.
+
+### Better Control Over Refresh Scheduling
+
+With SQA, you have more flexible scheduling options. You can set refresh intervals based on your specific requirements, ensuring efficient data management without overburdening driver nodes. This means greater control over resource usage and refresh frequency that better aligns with your needs.
+
+### Simplified Index Management
+
+SQA also simplifies index management by allowing you to adjust index settings without multiple, complicated queries. You can easily update refresh intervals or other parameters, thereby streamlining your workflow and reducing manual effort.
+
+## Getting Started
+
+### Prerequisites
+
```

Comment:
Also the document of prereq:

Step1:  for setup data source https://opensearch.org/docs/latest/dashboards/management/S3-data-source/, after then dashboard user can fire query via workbench, ref https://opensearch.org/docs/latest/dashboards/management/query-data-source/

Step2: user can accelerate query using secondary index ref https://opensearch.org/docs/latest/dashboards/management/accelerate-external-data/

During this process, user can change config to try out everything described on this doc